### ISSUE TITLE
Exception when try to move blocks within blockly div when blocks start shaking

### ISSUE THREAD

**Issue by ankuraga1508** [google/blockly#2929]
**Describe the bug**

Exception in the developer console when trying to move blocks after blocks start shaking

**To Reproduce**

1. Go to https://blockly-demo.appspot.com/static/demos/fixed/index.html
2. Open developer console
3. Drag blocks in the blockly area over each other until there is no space left or blocks start shaking.
4. Now remove any block by putting it back
5. Notice this exception : 
blockly_compressed.js:1802 Uncaught TypeError: Cannot read property 'getRootBlock' of null
    at blockly_compressed.js:1802
    at Blockly.WorkspaceSvg.Blockly.Workspace.fireChangeListener (blockly_compressed.js:1016)
    at Blockly.Events.fireNow_ (blockly_compressed.js:917)

**Expected behavior**

There should not be any error

![Capture](https://user-images.githubusercontent.com/15721876/63949623-50926580-ca40-11e9-869a-bbeb3b04d328.PNG)

**Video showing the steps**
[Video.zip](https://github.com/google/blockly/files/3556041/Video.zip)

ChromeOS : 74.0.3729.159 (Official Build) (64-bit)
Platform: 11895.118.0 (Official Build)
JavaScript: V8 7.4.288.28
Flash: 32.0.0.171

**Comment by RoboErikG** [comment]
Definitely a bug, but things seem to keep working after the error so lower priority to fix.

**Comment by ankuraga1508** [comment]
@RoboErikG Can we just handle the exception instead of throwing it?

**Comment by RoboErikG** [comment]
@ankuraga1508 that would just hide the bug without fixing it and could cause more issues later on.

**Comment by BeksOmega** [comment]
@NeilFraser I thought I fixed this when I fixed the shaking, but that must not have been the root problem. I can take a look in an hour or two.

**Comment by samelhusseini** [comment]
Source of the exception: 
https://github.com/google/blockly/blob/master/core/inject.js#L306

**Comment by samelhusseini** [comment]
Here are some simpler repro steps:
1. Load the Fixed demo: ``/demos/fixed/index.html``
2. Open the developer console, and run the following: 
```Blockly.Xml.domToWorkspace(Blockly.mainWorkspace, Blockly.Xml.textToDom(`<xml xmlns="https://developers.google.com/blockly/xml">
  <block type="controls_repeat_ext" x="11" y="0">
    <statement name="DO">
      <block type="controls_repeat_ext">
        <next>
          <block type="controls_repeat_ext">
            <next>
              <block type="controls_repeat_ext">
                <next>
                  <block type="controls_repeat_ext">
                    <next>
                      <block type="controls_repeat_ext">
                        <next>
                          <block type="controls_repeat_ext">
                            <statement name="DO">
                              <block type="controls_repeat_ext"/>
                            </statement>
                            <next>
                              <block type="controls_repeat_ext"/>
                            </next>
                          </block>
                        </next>
                      </block>
                    </next>
                  </block>
                </next>
              </block>
            </next>
          </block>
        </next>
      </block>
    </statement>
  </block>
</xml>`))
```
3. Drag the inner most repeat block into the flyout to delete it.
4. See exception in console.

**Comment by samelhusseini** [comment]
This was fixed by #3394
